app/Providers/ElasticsearchServiceProvider.php

```
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Providers;
+
+use Elastic\Elasticsearch\Client;
+use Elastic\Elasticsearch\ClientBuilder;
+use Illuminate\Support\ServiceProvider;
+
+class ElasticsearchServiceProvider extends ServiceProvider
+{
+    public function register(): void
+    {
+        $this->app->singleton(Client::class, function () {
+            return ClientBuilder::create()
+                ->setHosts([config('elasticsearch.host', 'http://localhost:9200')])
```

The default is set in the config, so if it's not a truly optional config item, the service should throw an exception if it's not set.  That strategy makes things safer against config items getting renamed or moved since it'll fail fast instead of falling back to a hardwired default.  I admit it's kind of a small nitpick in this particular case.